src/sentry/static/sentry/app/components/events/interfaces/frame/frame.tsx

```diff
@@ -281,15 +306,17 @@ export class Frame extends React.Component {
                 <PackageStatus status={this.packageStatus()} />
               </PackageLink>
             </PackageInfo>
-            <TogglableAddress
-              address={data.instructionAddr}
-              startingAddress={image ? image.image_addr : null}
-              isAbsolute={showingAbsoluteAddress}
-              isFoundByStackScanning={this.isFoundByStackScanning()}
-              isInlineFrame={this.isInlineFrame()}
-              onToggle={onAddressToggle}
-              maxLengthOfRelativeAddress={maxLengthOfRelativeAddress}
-            />
+            {data.instructionAddr && (
```

Comment:
I'm most concerned about creating a bug here. It seems like the address is mandatory but these kinds of changes have caused bugs before.

Reply:
yes, it makes sense to just render `TogglableAddress` is there is an address `data.instructionAddr` .

Reply:
This should be fine.